test_extras/test_chemprop/test_chemprop_pipeline.py

```diff
@@ -341,6 +347,16 @@ def test_prediction(self) -> None:
         self.assertEqual(proba.shape, proba_copy.shape)
         self.assertTrue(np.allclose(proba[~nan_indices], proba_copy[~nan_indices]))
 
+        # Test single prediction, this was causing an error before
+        single_mol_pred = classification_model.predict(
+            [molecule_net_bbbp_df["smiles"].iloc[0]]
+        )
+        self.assertEqual(single_mol_pred.shape, (1,))
+        single_mol_proba = classification_model.predict_proba(
+            [molecule_net_bbbp_df["smiles"].iloc[0]]
+        )
+        self.assertEqual(single_mol_proba.shape, (1, 2))
+
 
 class TestMulticlassClassificationPipeline(unittest.TestCase):
```

Comment:
wouldn't it make sense to test single predictions also in the Multiclass classification case?

Reply:
Good catch, that would have failed!
Fixed!